### ISSUE TITLE
UnicodeDecodeError when insert non unicode characters in presenter metadata field

### ISSUE THREAD

**Issue by dpeite** [teltek/Galicaster#588]
In media manager, when we edit a mediapackage info and insert characters like ñ, ó,...  in the presenter field occurs the next exception:

```
Traceback (most recent call last):
  File "/home/dpeite/Galicaster/galicaster/classui/listing.py", line 438, in on_edit
    self.edit(key)
  File "/home/dpeite/Galicaster/galicaster/classui/managerui.py", line 246, in edit
    self.repository.update(selected_mp)
  File "/home/dpeite/Galicaster/galicaster/mediapackage/repository.py", line 691, in update
    return self.__add(mp)
  File "/home/dpeite/Galicaster/galicaster/mediapackage/repository.py", line 807, in __add
    serializer.save_in_dir(mp, self.logger)
  File "/home/dpeite/Galicaster/galicaster/mediapackage/serializer.py", line 56, in save_in_dir
    m2.write(set_episode(mp)) #FIXME
  File "/home/dpeite/Galicaster/galicaster/mediapackage/serializer.py", line 464, in set_episode
    print unicode(mp.metadata_episode[name])
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1: ordinal not in range(128)
```

**Comment by Alfro** [comment]
Should be fixed on branches 2.0.x and higher

**Comment by silviabf** [comment]
Tested in release 2.0.2 and the bug exists in this version. Tested in branches 2.0.x and master and the bug is solved in this branches. I close this issue.
